### ISSUE TITLE
Cannot be installed in AIX 7.1

### ISSUE THREAD

**Issue by cloorc** [coder/code-server#7038]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### OS/Web Information

- Web Browser: Chrome 129.0.6668.71, 64bits
- Local OS: Windows 10
- Remote OS: AIX 7.1
- Remote Architecture: ppc64
- `code-server --version`:

### Steps to Reproduce

1. Login to AIX 7.1
2. Compile node 20.18.0 source code and install it
3. Install code-server with `npm install --global --verbose code-server`

### Expected

Can be installed successfully

### Actual

```text
npm error   g++-11 -shared -Wl,-bimport:/home/showcase/local/include/node/node.exp -pthread -Wl,-bbigtoc -maix64 -Wl,-blibpath:/usr/lib:/lib:/opt/freeware/lib/gcc/powerpc-ibm-aix7.3.0.0/11/pthread/ppc64:/opt/freeware/lib/gcc/powerpc-ibm-aix7.2.0.0/11/pthread/ppc64:/opt/freeware/lib/pthread/ppc64 -Wl,--gc-sections  -o Release/obj.target/argon2.node Release/obj.target/argon2/argon2_node.o Release/obj.target/argon2.a
npm error gmake: Leaving directory '/home/showcase/local/lib/node_modules/code-server/node_modules/argon2/build'
npm error ld: 0706-012 The -- flag is not recognized.
npm error ld: 0706-012 The -g flag is not recognized.
npm error ld: 0706-012 The -c flag is not recognized.
npm error ld: 0706-012 The -- flag is not recognized.
npm error collect2: error: ld returned 255 exit status
npm error gmake: *** [argon2.target.mk:153: Release/obj.target/argon2.node] Error 1
npm verbose cwd /home/sidev1
npm verbose os AIX 7.1
npm verbose node v20.18.0
npm verbose npm  v10.8.2
npm verbose exit 1
npm verbose code 1
```

### Logs

As previous posted

### Does this bug reproduce in native VS Code?

I did not test native VS Code

### Does this bug reproduce in GitHub Codespaces?

I did not test GitHub Codespaces

### Are you accessing code-server over a secure context?

- [x] I am using a secure context.

### Notes

There are two known issues and related PRs are being working on:

[ranisalt/node-argon2](https://github.com/ranisalt/node-argon2/issues/430) and [PR](https://github.com/ranisalt/node-argon2/pull/431)

[microsoft/vscode-policy-watcher](https://github.com/microsoft/vscode-policy-watcher/issues/34) and [PR](https://github.com/microsoft/vscode-policy-watcher/pull/35)

Upgrade of dependencies `argon2`(in `package.json`) and `@vscode/policy-watcher`(in `lib/vscode/package.json`) might solve this issue.

**Comment by benz0li** [comment]
@cloorc You may try the *unofficial* and *untested* Linux/ppc64le builds at https://gitlab.b-data.ch/coder/code-server/-/releases.

These builds are created using the multi-arch docker image [`glcr.b-data.ch/coder/code-server-builder`](https://gitlab.b-data.ch/coder/code-server-builder/container_registry).

Cross reference:

* https://github.com/coder/code-server/issues/6986#issuecomment-2335104284

**Comment by cloorc** [comment]
> [@cloorc](https://github.com/cloorc) You may try the _unofficial_ and _untested_ Linux/ppc64le builds at https://gitlab.b-data.ch/coder/code-server/-/releases.
> 
> These builds are created using the multi-arch docker image [`glcr.b-data.ch/coder/code-server-builder`](https://gitlab.b-data.ch/coder/code-server-builder/container_registry).
> 
> Cross reference:
> 
> * [Official RISC-V 64-bit release assets #6986 (comment)](https://github.com/coder/code-server/issues/6986#issuecomment-2335104284)

It seems not work as expected:

```text
root@hamesitdev101:/opt/freeware/bin> dnf install /home/showcase/code-server-4.93.1-ppc64le.rpm
Last metadata expiration check: 1 day, 23:03:10 ago on Wed Oct 16 14:36:43 CST 2024.
Error:
 Problem: conflicting requests
  - package code-server-4.93.1-1.ppc64le does not have a compatible architecture
(try to add '--skip-broken' to skip uninstallable packages or '--nobest' to use not only best candidate packages)
```

**Comment by benz0li** [comment]
@cloorc Your architecture seems to be `ppc64` and not `ppc64le` (introduced with the POWER8).

**Comment by cloorc** [comment]
> [@cloorc](https://github.com/cloorc) Your architecture seems to be `ppc64` and not `ppc64le` (introduced with the POWER8).

Yeah, it's right! And it seems like there is no prebuilt binaries.

**Comment by ravisankar-PIO** [comment]
Any luck on building for ppc64 architecture in the future?

**Comment by cloorc** [comment]
I'm afraid not. Cause vscode doesn't support AIX yet. And it seems like there is no plan for that. Check it here https://github.com/microsoft/vscode/issues/231568 .
